tests/Examples/InputObject.hs

```diff
@@ -26,26 +25,21 @@ root :: Handler IO Query
 root = pure (\dogStuff -> pure (show dogStuff))
 
 -- $setup
--- >>> import qualified GraphQL.Internal.Encoder as Encode
--- >>> import GraphQL.Value (valueToAST)
+-- >>> import Data.Aeson (encode)
+-- >>> import GraphQL.Value.ToValue (ToValue(..))
+
+-- TODO: jml thinks it's a bit confusing to have `show` output in these
```

Comment:
The braces & quotes within the string within the JSON confuse me a little syntactically. Hard to explain beyond what I've already commented. Have just done the TODO in 88056e4. Let me know if a) that clarifies what I meant by the TODO, b) you think that makes the example more effective.